tests/test_playbook_runs.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import os
+import tempfile
+
+from subprocess import Popen
+
+from unittest import TestCase
+
+import pytest
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def get_inventory_content():
+    ipa_server_host = os.getenv("IPA_SERVER_HOST")
+    return "[ipaserver]\n{}".format(ipa_server_host).encode("utf8")
```

Comment:
Since you are creating an inventory file, you could have `ansible_user` and `ansible_password`, so that not only key-based login is possible.

Reply:
The user you can already set using `ANSIBLE_REMOTE_USER`. The password is indeed not possible yet. I'll add to the improvements section in the readme.